src/platform/src/Bridge/ElevenLabs/ElevenLabsResultConverter.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\AI\Platform\Bridge\ElevenLabs;
+
+use Symfony\AI\Platform\Exception\RuntimeException;
+use Symfony\AI\Platform\Model;
+use Symfony\AI\Platform\Result\BinaryResult;
+use Symfony\AI\Platform\Result\RawResultInterface;
+use Symfony\AI\Platform\Result\ResultInterface;
+use Symfony\AI\Platform\Result\TextResult;
+use Symfony\AI\Platform\ResultConverterInterface;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Contracts\HttpClient\ResponseInterface;
+
+/**
+ * @author Guillaume Loulier <personal@guillaumeloulier.fr>
+ */
+final readonly class ElevenLabsResultConverter implements ResultConverterInterface
+{
+    public function __construct(
+        private string $outputPath,
+    ) {
+        if (!class_exists(Filesystem::class)) {
+            throw new RuntimeException('For using ElevenLabs as platform, the symfony/filesystem package is required. Try running "composer require symfony/filesystem".');
+        }
+    }
+
+    public function supports(Model $model): bool
+    {
+        return $model instanceof ElevenLabs;
+    }
+
+    public function convert(RawResultInterface $result, array $options = []): ResultInterface
+    {
+        /** @var ResponseInterface $response */
+        $response = $result->getObject();
+
+        return match (true) {
+            str_contains($response->getInfo('url'), 'speech-to-text') => new TextResult($result->getData()['text']),
+            str_contains($response->getInfo('url'), 'text-to-speech') => $this->doConvertTextToSpeech($result),
```

Comment:
let's skip persisting the file please:

```suggestion
            str_contains($response->getInfo('url'), 'text-to-speech') => new BinaryResult($result->getObject()->getContent(), 'audio/mpeg'),
```

my reasoning:
* we return the binary data and developer can do with it what they want
* from `ResultConverterInterface` point of view, writing on the file system is an unexpected side effect
* if someone would use a file system abstraction we would mess with that or would need to take it into account